src/dropdown/list/dropdown-list.component.ts

```diff
@@ -163,6 +164,8 @@ export class DropdownList implements AbstractDropdownView, AfterViewInit, OnDest
 	 * item selection.
 	 */
 	@Input() type: "single" | "multi" = "single";
+
+	@HostBinding("class.bx--list-box--expanded") hostClass = true;
```

Comment:
I'm not sure we want this to be a `HostBinding` in this case ... Eventually we'll want to apply the `expanded` class conditionally, without removing the dropdown list from the DOM, so we can pick up on the opening animation Carbon supplies. Right now we have a pretty clear separation of concerns (the dropdown list doesn't need to know when it's open, just how to render the list) and I'd like to keep it that way if possible.

Everything else looks good, I'm just not sure this is the right approach for `expanded`!